Source/Fuse.GeoLocation/Fuse.GeoLocation.unoproj

```diff
@@ -2,6 +2,9 @@
   "Copyright": "Copyright (c) Fuse Open 2018",
   "Description": "Native Location",
   "Publisher": "Fuse Open",
+  "Package": {
```

Comment:
One more thing: I removed the `Package` properties from project files earlier today, but your PR resurrects the property in this project file. Can you remove these three lines (basically revert `Fuse.GeoLocation.unoproj`) while at it?

Reply:
Sure, why not...